server/polar/order/schemas.py

```diff
@@ -179,15 +180,23 @@ class Order(CustomFieldDataOutputMixin, MetadataOutputMixin, OrderBase):
 
 class OrderUpdateBase(Schema):
     billing_name: str | None = Field(
+        default=None,
         description=(
             "The name of the customer that should appear on the invoice. "
             "Can't be updated after the invoice is generated."
-        )
+        ),
     )
     billing_address: Address | None = Field(
+        default=None,
         description=(
             "The address of the customer that should appear on the invoice. "
             "Can't be updated after the invoice is generated."
+        ),
+    )
+    custom_field_data: dict[str, str | int | bool | datetime.datetime | None] | None = (
```

Comment:
We have `CustomFieldDataInputMixin` in `polar/custom_field/data.py` that should be used to include that field with all its characteristics in the model.